Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -335,8 +363,8 @@ namespace CGAL {
   CGAL_assertion(nb_facets == expected_nb_facets);
   CGAL_USE(nb_facets); CGAL_USE(expected_nb_facets);
 
-
-  internal::Mesh_3::split_in_polylines(graph, polylines, K());
+  CGAL::Mesh_3::Polyline_visitor<Point_3, Graph> visitor(polylines, graph);
+  CGAL::split_graph_into_polylines(graph, visitor);
```

Comment:
@lrineau Actually the graph is also copied internally in `split_graph_into_polylines` while the former implementation was doing the modification in-place. If that matters, another version of `split_graph_into_polylines` working in-place should be added.